feature-configs/cn-ran-overlays/all-in-one/example/sriov/sriov-network-node-policy-netdevice-mh.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-mh-dpdk-site-1-fqdn-worker1
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: vfio-pci
+  isRdma: false
+  nicSelector:
+    pfNames: ["ens1f1#0-0"]
```

Comment:
This is using one vf only. Is this what is expected? In this way, only one pod will be able to use the vf. @SchSeba wdyt?

Reply:
@josephdrichard @vitus133 getting back on this, this will expose only one vf.
Is this what we want?

Reply:
Yes, one VF is fine. No more pods to attach to it

Reply:
Thanks!